src/components/CheckoutPriceBox/CheckoutPriceBox.js

```diff
@@ -14,111 +15,96 @@ import {
   StyledPriceWrapper
 } from './CheckoutPriceBoxStyled';
 
-const CheckoutPriceBox = ({
-  isCouponApplied,
-  finalPrice,
-  discountAmount,
-  taxValue,
-  customerServiceFee,
-  paymentMethodFee,
-  customerCurrencySymbol,
-  offerPrice,
-  taxRate,
-  country,
-  t
-}) => (
-  <StyledPriceBox>
-    <StyledPriceBoxWrapper>
-      <StyledPriceWrapper>
-        <StyledLabel>{t('Price')}</StyledLabel>
-        <StyledOfferPrice>
-          {`${customerCurrencySymbol}${formatNumber(offerPrice)} `}
-          <span>{country === 'US' ? t('excl. Tax') : t('excl. VAT')}</span>
-        </StyledOfferPrice>
-      </StyledPriceWrapper>
+const CheckoutPriceBox = ({ t }) => {
+  const {
+    priceBreakdown: {
+      offerPrice,
+      discountAmount,
+      taxValue,
+      customerServiceFee,
+      paymentMethodFee
+    },
+    discount: { applied: isCouponApplied },
+    taxRate,
+    country,
+    totalPrice: finalPrice
+  } = useSelector(state => state.order.order);
+  const { customerCurrencySymbol } = useSelector(state => state.offer.offer);
```

Comment:
as we are taking the prices from the order, we should also take the current from the same resource, so customerCurrencySymbol should be replaced with state.order.order.currency

Reply:
need more information